src/main/cpp/v8/JNIV8Object.cpp

```diff
@@ -96,7 +98,9 @@ void JNIV8Object::makeWeak() {
     // they can be destroyed / gced from java at any time, and there can exist multiple
     if(_v8ClassInfo->container->type == JNIV8ObjectType::kWrapper || _jsObject.IsWeak()) return;
     _jsObject.SetWeak((void*)this, JNIV8Object::weakPersistentCallback, WeakCallbackType::kFinalizer);
-    _jsObject.MarkIndependent();
+
+    //TODO: replace deleted method.
+    //_jsObject.MarkIndependent();
 
```

Comment:
Also here.

Reply:
MarkIndependant basically "simplifies gc"; v8 has some feature called object groups which we are not using. Calling this tells v8 that there are no dependencys between objects with this handle. Also just used to speed up gc a little, can safely be removed. (https://stackoverflow.com/questions/16731246/v8persistent-markindependent-what-does-this-method-exactly-do)